### ISSUE TITLE
Migration to GitHub is not complete

### ISSUE THREAD

**Issue by Neustradamus** [roundcube/roundcubemail#9256]
Dear @roundcube team,

It will be nice to remove old parts like:
- https://sourceforge.net/p/roundcubemail/news/
- https://sourceforge.net/p/roundcubemail/wiki/
- https://sourceforge.net/p/roundcubemail/cvs/
- https://www.paypal.com/cgi-bin/webscr?item_name=Donation+to+Roundcube+Webmail&cmd=_donations&business=hello%roundcube.net

Can you add a "move" box like https://sourceforge.net/projects/handbrake for example.
- https://sourceforge.net/p/roundcubemail/

Please update the website with HTTPS:
- https://sourceforge.net/projects/roundcubemail/
- https://sourceforge.net/projects/roundcubemail/support

To disable/delete tabs, go here: https://sourceforge.net/projects/roundcubemail/
To add a new tab, go here: https://sourceforge.net/projects/roundcubemail/
To change informations, go here: https://sourceforge.net/p/roundcubemail/admin/overview
To export, go here: https://sourceforge.net/p/roundcubemail/admin/export

The new place can be added in:
- "Moved Project to", here: https://sourceforge.net/p/roundcubemail/admin/overview
- A new tab, go here: https://sourceforge.net/projects/roundcubemail/

Thanks in advance.

PS: I think the access needs to be added to another people too ;)
Same for OpenHub: https://openhub.net/p/roundcube

**Comment by alecpl** [comment]
That project page is marked with status "abandoned". I don't recall having admin access to it. @thomascube I think we should remove it, if possible.

**Comment by Neustradamus** [comment]
@alecpl: Yes there are bad informations, update is needed and the moved box too...

**Comment by Neustradamus** [comment]
Dear @roundcube team, @alecpl, @ChristophWurst, @thomascube, @pabzm,

Have you progressed on it?

Can you add the "moved box" and remove old parts?

THanks in advance.
